### DISCUSSION TITLE
Menu component inside Table td is always positioned in top left corner of browser stacked

### DISCUSSION THREAD

**Discussion by danielbettech** [mantinedev/mantine#2941] [Q&A]
After I upgrade to mantine 5 and refactor the Table and Menu components I got the Menu positioned in top left corner stacked.

In the example, click the tool icon to test it. There is one icon per row that triggers the Menú.
The Menu component is implemented in the getFormattedData() function.

Here is the example
[Codesandbox example](https://codesandbox.io/s/elastic-khayyam-wv6ymj?file=/src/App.js)

Thanks!

**Comment by rtivital** [comment]
Tabler icons do not support `ref`, to fix that, wrap it with a `span` – https://codesandbox.io/s/vibrant-christian-dp04k7

**Reply by danielbettech** [reply]
Thanks @rtivital, this solved part of the issue. 
When the menu is clicked and visible, the closeOnClickOutside (default: true) is not working properly. If I click outside the table (bottom section of the page) the menu won't disappear, only if I click in some other part of the page (inside the table component for example).
